### ISSUE TITLE
Invalid version. Must be a string. Got type "object".

### ISSUE THREAD

**Issue by tomo-kn** [chromaui/chromatic-cli#1045]
**Bug report**

CLI now fails.

I get the same error in v1 and v11.

```
TypeError: ✖ Failed to build Storybook
Invalid version. Must be a string. Got type "object".
    at new e (/home/runner/work/_actions/chromaui/action/v11/action/register.js:195:15818)
    at P4r (/home/runner/work/_actions/chromaui/action/v11/action/register.js:195:20804)
    at Object.X4r [as gte] (/home/runner/work/_actions/chromaui/action/v11/action/register.js:195:21705)
    at oFn (/home/runner/work/_actions/chromaui/action/v11/action/register.js:802:890)
    at aFn (/home/runner/work/_actions/chromaui/action/v11/action/register.js:802:1258)
    at yse.task (/home/runner/work/_actions/chromaui/action/v11/action/register.js:286:6159)
Error: non-zero exit code
```

**Comment by fpaz** [comment]
we just started getting this error in our CI pipeline as well.

**Comment by aimee-gm** [comment]
Same here. We have 4 chromatic projects in the same repo & this is only affecting one of them.

**Comment by hannahpx** [comment]
Hello, we have identifed this issue and we are currently investigating this here: https://status.chromatic.com/incident/428641

We will update our status page frequently once we resolve this, thank you all and apologies for any inconvenience in the meantime. 

Hannah
@ Chromatic

**Comment by hannahpx** [comment]
@aimee-gm @fpaz @tomo-kn 

Hi,

While we continue to investigate this and your workflow isn't being blocked, can you try setting your CLI version  chromaui/action@11.9.0 as a workaround for now and let us know if the errors persist on this version? Thank you

Best regards,
Hannah
@ Chromatic

**Comment by michalmo** [comment]
I did already set that as a workaround.

It works fine with chromaui/action@11.9.0 and also with chromaui/action@v11.10.0 but fails with chromaui/action@v11.10.1 and chromaui/action@latest

I also noticed that chromaui/action@11.9.0 shows Chromatic CLI 11.8.0 in the logs and chromaui/action@v11.10.0 shows Chromatic CLI 11.9.0 in the logs.

The same can be reproduced locally, chromatic package works for chromatic@<=11.10.0 and fails for chromatic@11.10.1

**Comment by aimee-gm** [comment]
@hannahpx yes `chromaui/action@v11.9.0` works

**Comment by tmeasday** [comment]
@tomo-kn @fpaz @aimee-gm @michalmo what version of Storybook are you running?

**Comment by michalmo** [comment]
I was running 8.1.9 but have upgraded to 8.3.0,  fails with both

**Comment by Joe-Degler** [comment]
Running into similar error.

When running chromatic locally, it seems like it runs into an infinite loop where it starts spitting out endless recursive storybook metadata.

Running `chromatic": "^11.3.0` and `@storybook "^8.1.1"`.

```
11:04:29.110  Collecting Storybook metadata
11:04:29.140  {"storybookV6error":{}}
11:04:29.230  {"configDir":".storybook","mainConfig":{"_exports":{"stories":{"type":"ArrayExpression","start":187,"end":246,....
```

**Comment by tmeasday** [comment]
We just published `chromatic@11.10.2` (and the same to the action) after reverting a PR we think was responsible. Can y'all let us know if the issue is resolved for you?

**Comment by Joe-Degler** [comment]
LG, built and publish succeeded first try.

**Comment by Joe-Degler** [comment]
Seems like I spoke too early. The build and published seemed to succeed, but it did the endless recursive thing at the very end again. Not sure if this is related to this issue then, or not. Will investigate.

**Comment by tmeasday** [comment]
@Joe-Deglar - seems like maybe a second issue. What exact versions do you have and do you know when that issue started happening?

**Comment by tmeasday** [comment]
Also the messages you posted above don't necessarily seem off, were there more after those?

**Comment by Joe-Degler** [comment]
I agree that they don't seem off, but what does seem off is that all that metadata is being dumped into the console, before I upgraded the chromatic dependency right at the start, and now right at the end, and for some reason it doesn't stop.

**Comment by michalmo** [comment]
@tmeasday working for me, thanks.

**Comment by tmeasday** [comment]
When I run with `--debug` and 8.3 I see a huge log line which I suspect has something to do with prebundling changes in 8.3 that we'll investigate. But it doesn't actually stop anything from happening, it just dumps a bunch of stuff into my terminal. Maybe your terminal is crashing or something?

**Comment by Joe-Degler** [comment]
I'm not actually passing `--debug` at all 🤔 

This is my build command:

`yarn chromatic: yarn dlx chromatic --project-token=<REDACTED> --only-changed --working-dir "next" --untraced "next"`
`build-storybook: "NEXT_ENV=production storybook build`

**Comment by Joe-Degler** [comment]
Ah yes, you're right. I apologize, that was created without me knowing. Checking with `debug: false`. ...

Okay, no CLI diarrhea there. That's good. I definitely know that with the previous version it bugged out before anything could actually progress, now it only does it at the very end which means it still succeeds even if it requires cancelling on my side.

If you have a point of contact, I can send you a video of it happening. (I'd rather not share it in a public forum).

**Comment by hannahpx** [comment]
hi @Joe-Degler you can reach out to us at support@chromatic.com or use our in-app chat - thank you

**Comment by codykaup** [comment]
This is resolved in [v11.10.2](https://github.com/chromaui/chromatic-cli/releases/tag/v11.10.2) so I'm going to close this out. Let us know if you continue to see issues with this!
